app/packages/klogs/src/components/common/sortFields.ts

```diff
@@ -0,0 +1,30 @@
+// TODO: make this configurable in plugin options
+/* eslint-disable sort-keys */
+const columnPriority: Partial<Record<string, number>> = {
```

Comment:
I wouldn't add this feature, since the default fields do not make much sense to display them on the top, since most of the time we already filter for these fields. Instead I would add the Textfield to filter the list of fields like discussed last time.